packages/smart-contract/src/network/api.ts

```diff
@@ -46,13 +44,13 @@ export class BlockberryApi {
         options
       );
       if (!response.ok) {
-        logger.error('response:', response);
+        console.error('response:', response);
```

Comment:
Why are you change this to `console.error`?

Reply:
If we want to use logger from framework, smart-contract package now require `@orochi-network/framework` and the user that use zkdb need to install it.